site/src/helpers/substitutions.ts

```diff
@@ -0,0 +1,52 @@
+import { ProgramRequirement } from '@peterportal/types';
+
+export const labelSubstitutions: Record<string, string> = {
+  Div: 'Division',
+  "Int'l": 'International',
+  US: 'United States',
+};
+
+export function expandAbbreviation(label: string): string {
+  let expandedLabel = label;
+
+  for (const [abbrev, substitution] of Object.entries(labelSubstitutions)) {
+    const regex = new RegExp(`\\b${abbrev}\\b`, 'gi');
+    expandedLabel = expandedLabel.replace(regex, substitution);
+  }
+
+  return expandedLabel;
+}
+
+export function findCommonLabelPrefix(labels: string[]): string {
+  if (labels.length === 0) return '';
+
+  const expandedLabels = labels.map(expandAbbreviation);
+
+  expandedLabels.sort();
+  /* remove all text after parentheses */
```

Comment:
I would do the replace in the `expandAbbreviation` function and potentially rename that function to "formatLabel"